erclient/client.py

```diff
@@ -1344,7 +1344,7 @@ async def _call(self, path, payload, method, params=None):
             response = await self._http_session.request(
                 method,
                 self._er_url(path),
-                json=payload,  # payload is automatically encoded as json data
+                json=payload if method in ["POST", "PUT", "PATCH"] else None,  # payload is automatically encoded as json data
```

Comment:
The caller may try to do a GET with a body, and that triggers a quite cryptic 400 from ER (I guess it comes from some LB or GW). So this defensive code is to avoid that and just discard the payload in that case.